src/material/chips/chip-set.ts

```diff
@@ -75,6 +77,9 @@ export class MatChipSet
   /** Role to use if it hasn't been overwritten by the user. */
   protected _defaultRole = 'presentation';
 
+  /** Default chip options. */
+  protected _defaultOptions = inject(MAT_CHIPS_DEFAULT_OPTIONS, {optional: true});
```

Comment:
This doesn't look like it's used anywhere in the chip set. It might be better to move it to the listbox so it's closer to where it's used. We can always move it back here if we start needing it.

Reply:
Done I moved all the `_defaultOptions` in this PR so that they're only accessible in the classes that use them.